f8a_report/stack_report_main.py

```diff
@@ -4,22 +4,33 @@
 from datetime import datetime as dt, timedelta
 from report_helper import ReportHelper
 from v2.report_generator import StackReportBuilder
+import os
+import requests
 
 logger = logging.getLogger(__file__)
 
+_INVOKE_API_WORKERS = os.environ.get("INVOKE_API_WORKERS", "1")
+_WORKER_ADMINISTRATION_REGION = os.environ.get("WORKER_ADMINISTRATION_REGION", "api")
+_INGESTION_SERVICE_HOST = os.environ.get("INGESTION_SERVICE_HOST", "bayesian-jobs")
+_INGESTION_SERVICE_PORT = os.environ.get("INGESTION_SERVICE_PORT", "34000")
+_INGESTION_SERVICE_ENDPOINT = "api/v1/ingest-epv"
+_APP_SECRET_KEY = os.getenv('APP_SECRET_KEY', 'not-set')
+
 
 def main():
     """Generate the daily stacks report."""
     r = ReportHelper()
     report_builder_v2 = StackReportBuilder(ReportHelper)
     today = dt.today()
-    start_date = (today - timedelta(days=1)).strftime('%Y-%m-%d')
+    start_date = (today - timedelta(days=500)).strftime('%Y-%m-%d')
```

Comment:
why is it 500?

Reply:
Oh Sorry, It was for my testing, will remove it.